boost.sh

```diff
@@ -850,6 +856,38 @@ scrunchAllLibsTogetherInOneLibPerPlatform()
     done
 }
 
+buildUniversalStatic()
```

Comment:
Again, since we're only building static libraries anyways, you can omit the 'static' bit: `buildUniversal()`